### PR TITLE
Feature/add recognition

### PR DESCRIPTION
**Pull request by Andresdev02**
CLBV-310
CLBV-194

Start the frontend with ``` ember s --proxy https://verenigingen.oscart-dev.s.redhost.be ```

### CODE REVIEW

File: app/components/recognition/form.hbs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                  </:label>
+                  <:content>
+                    <AuInput
+                      @value={{mut
+                        this.currentRecognition.recognitionModel.legalResource
+                      }}
+                      @width={{"block"}}
+                      id="erkenningsbesluit"
+                    />
+                    {{! <div class="au-u-flex au-u-flex--end">
```
Comment (lagartoverde) [inline app/components/recognition/form.hbs:110]: Remove commented out code if we don't need it

File: app/controllers/association/recognition/create.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import AssociationRecognitionEditController from './edit';
+
+export default class AssociationRecognitionCreateController extends AssociationRecognitionEditController {}
```
Comment (lagartoverde) [inline app/controllers/association/recognition/create.js:3]: this is weird, from controllers to extend each other, I would just repeat the code, in the future we might add things to the Edit controller that we don't want in the create

File: tests/unit/services/current-recognition-test.js
Comment (lagartoverde) [inline tests/unit/services/current-recognition-test.js:1]: Remove empty tests

**Review by lagartoverde — APPROVED** [review]
Looks good, some minor comments.
Just to make sure, this doesn't include any validation right? I set the end date before the start date and in the past and it didn't complain

**Comment by Andresdev02** [comment]
@lagartoverde thanks for your review. Indeed the validations are in another ticket.

**Review by lagartoverde — APPROVED** [review]

**Pull request merged**